README.md

```diff
@@ -157,6 +157,7 @@ itself) to look for plugins.
 | Operator SDK                  | [Medium/asdf-operator-sdk](https://github.com/Medium/asdf-operator-sdk)                           | [![Build Status](https://travis-ci.org/Medium/asdf-operator-sdk.svg?branch=master)](https://travis-ci.org/Medium/asdf-operator-sdk)                                                     |
 | Packer                        | [Banno/asdf-hashicorp](https://github.com/Banno/asdf-hashicorp)                                   | [![Build Status](https://travis-ci.org/Banno/asdf-hashicorp.svg?branch=master)](https://travis-ci.org/Banno/asdf-hashicorp)                                                             |
 | PHP                           | [asdf-community/asdf-php](https://github.com/asdf-community/asdf-php)                             | [![Build status](https://github.com/asdf-community/asdf-php/workflows/Main%20workflow/badge.svg?branch=master)](https://github.com/asdf-community/asdf-php/actions)                     |
+| Pluto                         | [FairwindsOps/pluto](https://github.com/asdf-pluto)                                               | [![GitHub Actions Status](https://github.com/FairwindsOps/asdf-pluto/workflows/build/badge.svg?branch=master)](https://github.com/FairwindsOps/asdf-pluto/actions)                      |
```

Comment:
This URL 404s - `https://github.com/asdf-pluto`.

Reply:
Oops...

Reply:
Fixed in latest master